src/acore-wp-plugin/src/Components/UserPanel/UserController.php

```
@@ -1,175 +1,224 @@
+    /**
+     * @return UserView
+     */
+    public function getView() {
+        return $this->view;
+    }
+
+    /**
+     * @return UserModel
+     */
+    public function getModel() {
+        return $this;
```

PHPStan (2.2.2)

[error] Method ACore\Components\UserPanel\UserController::getModel() should return ACore\Components\UserPanel\UserModel but returns $this(ACore\Components\UserPanel\UserController).


(return.type)